src/PerfView/StackViewer/PerfDataGrid.xaml.cs

```diff
@@ -245,6 +245,12 @@ private string CompressContent(string content)
                 return content;
             }
 
+            // Check if the user option is set to not truncate/shorten content
+            if (bool.TryParse(App.UserConfigData["DoNotTruncateStackViewerLinesOnPaste"], out bool doNotTruncateLines) && doNotTruncateLines)
```

Comment:
It would be good to cache the parsing here since this is going to be called for each stack frame.  Perhaps this can be cached in the initialize and toggle methods?  It would be reasonable to have a static variable in the `PerfDataGrid` class.

Reply:
@brianrob in thinking more about this, I do like the ability to be able to flip the option while the stacks window is already open, and have it immediately pick the change up. That way if a set of frames are already the way I want but I want to paste them a specific way, then I don't have to back out and reload the stacks window and refilter to have to do it. If I do it in the Initialize and Toggle methods then it will only be set during instantiation of the window, so changing it would require reloading the window if I'm not mistaken. 
My thought to keep this functionality is to instead place the check (and update a static class variable) in PerfDataGrid.xaml.cs at the beginning of the Grid.CopyingRowClipboardContent anonymous delegate, right above the for() loop. This was it's checked every time content is copied, but not for every stack frame.
Thoughts?

Reply:
@brianrob this has been implemented and pushed (along with some naming changes), please review - thanks!